source/compatibility.txt

```diff
@@ -0,0 +1,166 @@
+application uses ActiveSupport or Ruby on Rails, you must load the driver's
+ActiveSupport compatibility code for behavior such as time serialization to be
+correct:
+
+.. code-block:: ruby
+
+   require 'mongo'
+   require 'mongo/active_support'
+
+Applications using Mongoid 7.0.6 or newer do not need to explicitly load
+the driver's ActiveSupport code, since Mongoid automatically does so.
+
+.. _tls-compatibility:
+
+TLS/SSL Compatibility
+----------------------
+
+The {+driver-short+} uses the protocols supported by the underlying Ruby
+``openssl`` extension. The ``openssl`` extension generally exposes
+the functionality available in the operating system's OpenSSL library.
+
+Industry best practices, and some regulations, require the use of TLS 1.1
+or newer. Some operating systems or versions may not provide an OpenSSL version
+that supports these TLS versions.
+
+If you use macOS older than 10.13 (High Sierra), you need to install Ruby from
+`rvm`_, `homebrew`_, `macports`_, or another similar source. See
+`Installing Ruby`_ for more options.
+
+If you use Linux or other non-macOS Unix, you can check your OpenSSL version
+as follows:
+
+.. code-block:: sh
+
+  openssl version
+
+If the version number is less than 1.0.1, support for TLS 1.1 or newer is
+not available. Contact your operating system vendor for a solution or upgrade
+to a newer distribution.
+
+You can check your Ruby interpreter by running the following command:
+
+.. code-block:: sh
+
+  ruby -e "require 'net/http'; require 'json'; puts JSON.parse(Net::HTTP.get(URI('https://www.howsmyssl.com/a/check')))['tls_version']"
+
+You should see ``TLS 1.X`` where ``X`` is greater than or equal to ``1``.
+
+To learn more about TLS versions and their security implications, see `Transport Layer Security Cheat Sheet
+<https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Security_Cheat_Sheet.html>`_.
+
+.. _rvm: https://rvm.io/
+.. _homebrew: https://brew.sh/
+.. _macports: https://www.macports.org/
+.. _Installing Ruby: https://www.ruby-lang.org/en/documentation/installation
+
+Atlas Compatibility
+-------------------
+
+We recommend `Driver version 2.6.1
+<https://github.com/mongodb/mongo-ruby-driver/releases/tag/v2.6.1>`_ or higher
+when using MongoDB Atlas. This version has significant performance improvements
+for TLS connections, and all Atlas connections use TLS.
+
+When running on JRuby and connecting to Atlas Free Tier,
+`driver version 2.6.4 <https://github.com/mongodb/mongo-ruby-driver/releases/tag/v2.6.4>`_
+or higher and Java 8 or higher are required.
+
+mongo_kerberos Compatibility
+----------------------------
+
+The following compatibility table specifies the version(s) of the
+:ref:`mongo_kerberos library <kerberos>` to use with a specific version of
+the {+driver-short+}.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Ruby Driver
+     - mongo_kerberos |nbsp| 2.1
```

Comment:
I: `|nbsp|` seems to not be rendering, causing an extra space.

Also, give that the table is so short, would this section be better served by a paragraph?

Reply:
I think this is just a non-breaking space so nothing should render. It was present in the original compatibility page but I can update this to no longer use a table and remove it if not needed.